src/app/utils/route-utils.ts

```diff
@@ -72,6 +72,12 @@ export abstract class RouteUtils {
     return RouteUtils.ENABLED_LAYERS_FOR_NETWORK[network] || []
   }
 
+  static getEnabledScopes(): { network: Network; layer: Layer }[] {
+    return RouteUtils.getEnabledNetworks().flatMap(network =>
```

Comment:
Can we have a readonly const at the top of the file instead

Reply:
Yes, but this part is already being touched by the big search overhaul PR, so I'll do it there instead.